cypress/e2e/accessibility.cy.ts

```diff
@@ -24,6 +24,13 @@ describe("Accessibility tests in light mode", () => {
         cy.checkAccessibility();
     });
 
+    it("Checks add-clients page", () => {
+        cy.visit("/add-clients");
```

Comment:
is it worth making this /clients/add instead? That makes a bit more semantic sense.